npm/tailcall/scripts/generate-packages.js

```diff
@@ -0,0 +1,66 @@
+import * as fs from "node:fs"
+import {dirname, resolve} from "node:path"
+import {fileURLToPath} from "node:url"
+
+const APP_VERSION = process.env.APP_VERSION
+
+const MANIFEST_PATH = resolve(fileURLToPath(import.meta.url), "../..", "package.json")
```

Comment:
```suggestion
const MANIFEST_PATH = resolve(fileURLToPath(import.meta.url), "../package.json")
```

shouldn't it be like this? as package.json is jus one level above

Reply:
Yes, fixed.